src/components/cc-logs-controller/cc-logs-controller.stories.js

```diff
@@ -0,0 +1,59 @@
+import './cc-logs-controller.js';
+import { makeStory } from '../../stories/lib/make-story.js';
+import { enhanceStoriesNames } from '../../stories/lib/story-names.js';
+import { createFakeLogs, CUSTOM_METADATA_RENDERERS } from '../cc-logs/fake-logs.js';
+
+export default {
+  title: '🚧 Beta/🛠 Logs/<cc-logs-controller-beta>',
+  component: 'cc-logs-controller-beta',
+};
+
+const conf = {
+  component: 'cc-logs-controller-beta',
+  beta: true,
+  // language=CSS
+  css: `
+    cc-logs-controller-beta {
+      border: 1px solid #ddd;
+      border-radius: 0.2em;
```

Comment:
issue: I'm not sure we want to display such a border, it may be confusing for reviewers, thinking it's part of the component